shared/AppInsightsCore/src/JavaScriptSDK.Enums/TelemetryUnloadReason.ts

```diff
@@ -13,15 +13,15 @@
     /**
      * Just this plugin is being removed
      */
-    //PluginUnload = 1,
+    PluginUnload = 1,
 
     /**
      * This instance of the plugin is being removed and replaced
      */
-    //PluginReplace = 2,
+    PluginReplace = 2,
 
     /**
      * The entire SDK is being unloaded
      */
-    //SdkUnload = 50
+    SdkUnload = 50
```

Comment:
Big jump from 2 to 50, any specific reason for this one?

Reply:
Yeh, I figured SDK unload should be separated from the other "plugin" related reasons